tools/include/compiler_options.hpp.in

```diff
@@ -948,9 +948,13 @@ static Options CreateOptions(bool add_defaults=true) {
 
 #endif
 
+   // 1.2 -- action results
+   // 1.3 -- sync calls, bitset type, action and sync call names in C++
+   //        identifier format
+
    /* TODO add some way of defaulting these to the current highest version */
    int abi_version_major = 1;
-   int abi_version_minor = 2;
+   int abi_version_minor = 3;
```

Comment:
I think the abi version should be updatyed to 1.3 only if sync calls are used https://github.com/AntelopeIO/cdt/issues/355

Reply:
As this is on `sync_call` branch only, for now I just keep it as is. When it is merged into `main`, it will be updated based on how #355 handles versions.